compiler/lib/javascript.ml

```diff
@@ -116,7 +116,8 @@ and property_name_and_value_list = (property_name * expression) list
 and property_name =
   | PNI of identifier
   | PNS of string
-  | PNN of float
+  | PNInt of Int64.t
```

Comment:
Should that be `Int32.t`? JavaScript does not have 64-bit integers.

Reply:
This was initially to represent values that look like integers, which could include numbers that are much higher than 32-bit, but which still looked like integers in the source.  (Not that it matters anymore, the changes that Hugo and I talked about and which he implemented is a much better design IMO)